### PR TITLE
More xrefs and some cleanup for manual

### PR DESCRIPTION
**Pull request by kshyatt**
This passed `make docs` locally.

### CODE REVIEW

File: doc/src/manual/modules.md [hunk 1]
```diff
@@ -119,25 +119,25 @@ end
...
 
-Base is a module that contains basic functionality (the contents of base/). All modules implicitly contain `using Base`,
+Base is a module that contains basic functionality (the contents of `base/`). All modules implicitly contain `using Base`,
 since this is needed in the vast majority of cases.
 
 ### Default top-level definitions and bare modules
 
 In addition to `using Base`, modules also automatically contain
-definitions of the `eval` and `include` functions,
+definitions of the [`eval`](@ref) and [`include`](@ref) functions,
```
Comment (fredrikekre) [inline doc/src/manual/modules.md]: While you are at it; I think this needs to be corrected for https://github.com/JuliaLang/julia/pull/27098

**Review by fredrikekre — APPROVED** [review]
Looks good, but see comment about `eval`.

**Comment by kshyatt** [comment]
dang ci is unhappy. i looked at a few of the logs and it doesn't seem like most of these are my fault?

**Comment by kshyatt** [comment]
Lol what the heck. I just pushed to it! Anyway, hello from my new work computer.

**Comment by kshyatt** [comment]
Oh well, goodbye to this Hell PR I guess.

**Comment by timholy** [comment]
Hopefully the work ends up elsewhere? Sorry it was trouble.

**Pull request closed without merging**